src/editor.cpp

```diff
@@ -1004,25 +1004,33 @@ void Editor::setCursorRectVisible(bool visible) {
         ui->graphicsView_Map->scene()->update();
 }
 
+bool Editor::isWithinMap(int x, int y) {
```

Comment:
nitpick: It feels like like bounds-checking method should belong in the `Map` class, rather than `Editor`.